acceptance/README.md

```diff
@@ -1,257 +1,450 @@
+is a known issue, and we are working to make this a better experience for our
+community.
 
-The rake tasks for running the tests are defined by the Rakefile in the acceptance test directory.
-These tasks come with some documentation: `rake -T` will give short descriptions, and a `rake -D` will give full descriptions with information on ENV options required and optional for the various tasks.
-
-If you are setting up a new repository for acceptance, you will need to bundle install first.  This step assumes you have ruby and the bundler gem installed.
+-------------
 
-```sh
-cd /path/to/repo/acceptance
-bundle install --path=.bundle/gems
+## Setup
+### Prerequisites
+This assumes you have git, ruby, and the [bundler][] gem installed. You'll need
+a local clone of the puppet repo. All command examples in this readme assume you
+are working in the acceptance directory, where this README is located.
+
+### Installation
+All of the dependencies you need to run and develop tests are defined in
+`Gemfile`. To install them, run `bundle install --path .bundle/gems`. This
+command, as will all the command examples in this README, assume you are working
+in the acceptance directory. If you ever have issues with your runtime
+dependencies, you can update them with `bundle update`, or start over fresh with
+`rm -rf .bundle/gems; bundle install`.
+
+To ensure installation was successful, you can run `bundle exec rake -T`. This
+should return something along these lines:
 ```
-
-### Using Git Mirrors
-
-By default if you are installing from source, packages will be installed from Github, from their puppetlabs forks.  This can be selectively overridden for all installed projects, or per project, by setting environment variables.
-
-* SERVER => this will be the address the git server used for all installed projects.  Defaults to 'github.com'.
-* FORK => this will be the fork of the project for all installed projects.  Defaults to 'puppetlabs'.
-
-To customize the server or fork for a specific project use PROJECT_NAME_SERVER and PROJECT_NAME_FORK.
-
-For example, run with these options:
-
-```sh
-bundle exec rake ci:test:git CONFIG=config/nodes/win2008r2.yaml SHA=abcd PUPPET_SERVER=percival.corp.puppetlabs.net SERVER=github.delivery.puppetlabs.net
+$ bundle exec rake -T
+rake ci:help               # Print usage information
+rake ci:test:aio           # Run the acceptance tests using puppet-agent (AI...
+rake ci:test:gem           # Run the acceptance tests against puppet gem on ...
+rake ci:test:git           # Run the acceptance tests against a git checkout
+rake ci:test:quick         # Run a limited but representative subset of acce...
+rake clean                 # Remove any temporary products
+rake clobber               # Remove any generated files
 ```
+To get a detailed description of all of these tasks, run `bundle exec rake -D`.
 
-Beaker will install the following:
-
+-------------
+## Quick Start
+### For community members
```

Comment:
@smcclellan @jtappa I added a quick start section, but it's sad for community members :(

Reply:
I'm sorry that it's disappointing